js-exercises/least-common-multiple/leastCommonMultiple.js

```diff
@@ -0,0 +1,33 @@
+
+function leastCommonMultiple(...args) {
```

Comment:
More simple logic can be used to find the least common multiple.
Like in the question it was asked that least common of two given number. Like leastCommonMultiple(6,21). So the least common multiple will be greater than the larger number in the given two. So just check that larger number multiple and check if that is divisible by the smaller number then that will be the least common multiple.